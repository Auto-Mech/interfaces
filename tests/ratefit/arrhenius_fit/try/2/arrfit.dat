 1.0e-10 1.0e-10 
 1 1 1 1 1 1 
 1 
   8.1e-11, -0.01, 2000.0
 5   5 
600.0     9.357E+01
700.0     2.287E+03
800.0     2.710E+04
900.0     1.933E+05
1000.0    9.558E+05
