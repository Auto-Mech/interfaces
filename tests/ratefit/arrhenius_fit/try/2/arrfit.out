 initial guess
    8.10000E-011  -1.00000E-002   2.00000E+003
   600.00      93.570     0.28694E-11  100.00    
   700.00      2287.0     0.46125E-11  100.00    
   800.00      27100.     0.65835E-11  100.00    
   900.00     0.19330E+06 0.86813E-11  100.00    
   1000.0     0.95580E+06 0.10830E-10  100.00    
 results for iteration           1
 chi  0.28786E-01lamda  0.10000E-03
 params
    8.61016E-002   3.79509E+000   1.02160E+004
    5.18513E+022   3.79509E+000   2.02992E+004
   600.00      93.570      121.27      29.606    
   700.00      2287.0      2478.6      8.3763    
   800.00      27100.      25501.      5.8993    
   900.00     0.19330E+06 0.16478E+06  14.754    
   1000.0     0.95580E+06 0.76477E+06  19.986    
 results for iteration           2
 chi  0.68359E-04lamda  0.10000E-04
 params
    1.31608E-001   3.86567E+000   1.08931E+004
    7.92559E+022   3.86567E+000   2.16445E+004
   600.00      93.570      94.196     0.66940    
   700.00      2287.0      2286.7     0.13078E-01
   800.00      27100.      26802.      1.0983    
   900.00     0.19330E+06 0.19185E+06 0.74887    
   1000.0     0.95580E+06 0.96715E+06  1.1878    
 results for iteration           3
 chi  0.53781E-04lamda  0.10000E-05
 params
    2.45628E-001   3.78463E+000   1.09578E+004
    1.47919E+023   3.78463E+000   2.17731E+004
   600.00      93.570      93.986     0.44429    
   700.00      2287.0      2288.2     0.54253E-01
   800.00      27100.      26840.     0.95929    
   900.00     0.19330E+06 0.19201E+06 0.66491    
   1000.0     0.95580E+06 0.96667E+06  1.1368    
 results for iteration           4
 chi  0.21823E-04lamda  0.10000E-06
 params
    2.33053E+000   3.49065E+000   1.11820E+004
    1.40347E+024   3.49065E+000   2.22187E+004
   600.00      93.570      93.580     0.10896E-01
   700.00      2287.0      2296.8     0.43017    
   800.00      27100.      26962.     0.50881    
   900.00     0.19330E+06 0.19222E+06 0.55986    
   1000.0     0.95580E+06 0.96184E+06 0.63226    
 results for iteration           5
 chi  0.17036E-04lamda  0.10000E-07
 params
    7.77680E+000   3.33320E+000   1.13021E+004
    4.68327E+024   3.33320E+000   2.24573E+004
   600.00      93.570      93.364     0.22043    
   700.00      2287.0      2301.5     0.63209    
   800.00      27100.      27028.     0.26672    
   900.00     0.19330E+06 0.19233E+06 0.50356    
   1000.0     0.95580E+06 0.95927E+06 0.36307    
 results for iteration           6
 chi  0.17023E-04lamda  0.10000E-08
 params
    8.32242E+000   3.32434E+000   1.13089E+004
    5.01184E+024   3.32434E+000   2.24707E+004
   600.00      93.570      93.352     0.23343    
   700.00      2287.0      2301.7     0.64347    
   800.00      27100.      27031.     0.25308    
   900.00     0.19330E+06 0.19233E+06 0.50039    
   1000.0     0.95580E+06 0.95913E+06 0.34794    
 results for iteration           7
 chi  0.17023E-04lamda  0.10000E-09
 params
    8.32561E+000   3.32429E+000   1.13089E+004
    5.01377E+024   3.32429E+000   2.24708E+004
   600.00      93.570      93.352     0.23350    
   700.00      2287.0      2301.7     0.64353    
   800.00      27100.      27031.     0.25300    
   900.00     0.19330E+06 0.19233E+06 0.50037    
   1000.0     0.95580E+06 0.95912E+06 0.34786    
